### DISCUSSION TITLE
Count issue with policy rules

### DISCUSSION THREAD

**Discussion by rafpor75** [mia-platform/community#569] [Q&A] [answered]
### Description

Which section?
micro-lc / composer

Which is the version?
2.2.2

Description
Hi,

I created a rule to filter the response records of a CRUD service call, specifically for the endpoint /main. Without the filter, I receive 100 records in the response; with the filter applied, I get 75 records. When I display the results through the back office, it shows the filtered records, but the count reflects the total number of unfiltered records. I noticed that the count query returns the total number of records as well. How can I return the count of only the filtered records?

thank you

**Comment by GabrieleGuelfi** [comment, marked as answer]
Hello @rafpor75,
you should add the filter to the `/main/count` route as well.

**Reply by rafpor75** [reply]
thank you @GabrieleGuelfi it works ;)
